fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingSubspaces.java

```diff
@@ -103,10 +103,24 @@ public static Subspace indexScrubRecordsRangeSubspaceZero(@Nonnull FDBRecordStor
      * @return subspace
      */
     @Nonnull
-    public static Subspace indexScrubRecordsRangeSubspace(@Nonnull FDBRecordStoreBase<?> store, @Nonnull Index index) {
+    private static Subspace indexScrubRecordsRangeSubspaceRoot(@Nonnull FDBRecordStoreBase<?> store, @Nonnull Index index) {
         return indexBuildSubspace(store, index, INDEX_SCRUBBED_RECORDS_RANGES);
     }
 
+    /**
+     * Subspace that stores scrubbed records ranges. This subspace is expected to be followed by a range-id.
```

Comment:
The second half of this comment doesn't seem right. 
```suggestion
     * Subspace that stores scrubbed records ranges.
```

Reply:
✅